src/vicsetup/libvicsetup/defs.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _VIC_DEFS_H
+#define _VIC_DEFS_H
+
+#define VIC_INLINE static __inline__
+
+#define VIC_PACK_BEGIN _Pragma("pack(push, 1)")
+#define VIC_PACK_END _Pragma("pack(pop)")
+
+#define VIC_COUNTOF(ARR) (sizeof(ARR) / sizeof(ARR[0]))
+
+#define VIC_OFFSETOF(TYPE, MEMBER) __builtin_offsetof(TYPE, MEMBER)
+
+#define VIC_UNUSED __attribute__((unused))
+
+#define __LUKS_CONCAT(X, Y) X##Y
```

Comment:
Why is this called `__LUKS_CONCAT` not `__VIK_CONCAT`?

Symbols that start with a double underscore are reserved for the implementation in C, so defining a macro that starts with double underscores is undefined behaviour, please change this to a valid C identifier.